README.md

```diff
@@ -299,8 +300,34 @@ Alternatively, create an `environment.local.py` file in the project root
 directory and specifiy a global default for `GOOGLE_APPLICATION_CREDENTIALS`
 there.
 
+### 2.3.3 OAuth Client Credentials
 
-### 2.3.3 Google Cloud, TDR and SAM
+In order for users to authenticate using OAuth, the deployment must be
+registered as an OAuth client.
+
+1. Log into the Google Cloud console (same project as the previous section)
+
+2. Navigate to *APIs & Services* -> *Credentials*; click *+ CREATE CREDENTIALS* -> *OAuth Client ID*
+
+3. For *Application Type*, select *Web application*
+
+4. For *Name*, ``_select`` your deployment, evaluate ``azul.config.service_name`` in python, and enter the result
```

Comment:
I think you can assume that the target deployment is selected (or add instructions to do so as the first step), and provide the exact python -c command line, like `python -c 'from azul import config; print(config.service_name)'`